src/Plugins/ITKImageProcessing/src/ITKImageProcessing/ITKImageProcessingPlugin.cpp

```diff
@@ -34,7 +34,12 @@ ITKImageProcessingPlugin::ITKImageProcessingPlugin()
   {
     addFilter(filterFunc);
   }
-  RegisterITKImageIO();
+
+  if(!isRegistered)
+  {
+    RegisterITKImageIO();
+    isRegistered = true;
+  }
```

Comment:
```suggestion
  static bool s_IsRegistered = false;
  if(!s_IsRegistered)
  {
    RegisterITKImageIO();
    s_IsRegistered = true;
  }
```
I would suggest moving the bool inside the function since it is only relevant to this spot.